installation/on_debian.rst

```diff
@@ -53,6 +53,27 @@ reconfiguring the network using the :command:`omv-firstaid` command.
     systemctl enable --now systemd-resolved.service
     resolvectl dns <INTERFACE> <DNS_SERVER_IP>
 
+If the last step fails, you can fix it by manually adding some DNS servers to the
+`systemd-resolved` configuration. Of course you can use any other DNS server(s)
+of your choice.
+
+.. code-block:: console
+
+    mkdir -p /etc/systemd/resolved.conf.d
+    echo -e '[Resolve]\nDNS=8.8.8.8' > /etc/systemd/resolved.conf.d/mydns.conf
```

Comment:
This will later cause problems when OMV builds the network configuration via netplan.

I have not tested that, but the mentioned `resolvectl dns <INTERFACE> <DNS_SERVER_IP>` should do the same as creating a configuration file. 
The temporary solution might not cause problems/failures later when the systemd network configuration is build via netplan.